src/main/resources/digital/slovensko/autogram/server/server.yml

```diff
@@ -527,12 +527,41 @@ components:
         schema:
           type: string
           example: '<?xml version="1.0"?><xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"><xs:element name="Document"><xs:complexType><xs:sequence><xs:element name="Title" type="xs:string" /></xs:sequence></xs:complexType></xs:element></xs:schema>'
-          description: Optional XML schema. Format is dictated by `payloadMimeType`. Is overridden by autoLoadEform.
+          description: Optional XML schema used to validate the signing document and to compute digest used in "UsedXSDReference" in "DigestValue" attribute inside created XML Datacontainer. Format (plaintext or base64) is dictated by `payloadMimeType`. Is overridden by autoLoadEform.
+
+        schemaIdentifier:
+          type: string
+          example: http://schemas.gov.sk/form/App.GeneralAgenda/1.9/form.xsd
+          description: Optional identifier of the XML schema. The value is used in "UsedXSDReference" field inside created XML Datacontainer. Schema must be provided with this parameter. Is overridden by autoLoadEform. Default value is "http://schemas.gov.sk/form/<form-idnetifier>/<version>/form.xsd".
 
         transformation:
           type: string
           example: '<?xml version="1.0"?><xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"><xsl:template match = "/"><h1><xsl:value-of select="/Document/Title"/></h1></xsl:template></xsl:stylesheet>'
-          description: Optional XML transformation used to . Format is dictated by `payloadMimeType`. Is overridden by autoLoadEform.
+          description: Optional XML transformation used to present the signing document to user and to compute digest used in "UsedPresentationSchemaReference" in "DigestValue" attribute inside created XML Datacontainer. Format (plaintext or base64) is dictated by `payloadMimeType`. Is overridden by autoLoadEform.
+
+        transformationIdentifier:
+          type: string
+          example: http://schemas.gov.sk/form/App.GeneralAgenda/1.9/form.xslt
+          description: Optional identifier of the XML transformation. The value is used in "UsedPresentationSchemaReference" field inside created XML Datacontainer. Transformation must be provided with this parameter. Is overridden by autoLoadEform. Default value is "http://schemas.gov.sk/form/<form-idnetifier>/<version>/form.xslt".
+
+        transformationLanguage:
+          type: string
+          example: sk
+          description: Optional language of the XML transformation. The value is used in "UsedPresentationSchemaReference" in "Language" attribute inside created XML Datacontainer. Transformation must be provided with this parameter. Is overridden by autoLoadEform. Default value is "sk".
+
+        transformationMediaDestinationTypeDescription:
+          type: string
+          enum:
+            - TXT
+            - HTML
+            - XHTML
```

Comment:
Toto teraz berieme z transformacie, ak sa to da "overridnut" alebo to v transformacii nemusi byt, tak to potrebujeme vediet.